src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -42,6 +44,15 @@ public static string GetOperationTree(IOperation operation, int initialIndent =
             return walker._builder.ToString();
         }
 
+        public static void Verify(string expectedOperationTree, string actualOperationTree)
+        {
+            char[] newLineChars = Environment.NewLine.ToCharArray();
+            string actual = actualOperationTree.Trim(newLineChars);
+            expectedOperationTree = expectedOperationTree.Trim(newLineChars);
+            expectedOperationTree = Regex.Replace(expectedOperationTree, "([^\r])\n", "$1" + Environment.NewLine);
```

Comment:
This is due to the CDATA for VB. It seems to have `\n` characters instead of new lines, and this causes equality check to fail. This line replaces `\n` characters, that do not have a `\r` before it, with `\r\n'